packages/manager/apps/pci-public-ip/tailwind.config.js

```diff
@@ -0,0 +1,6 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ['./src/**/*.{js,jsx,ts,tsx}'],
+  theme: {},
+  plugins: [],
```

Comment:
tailwind config should be updated to disable preflight and include manager-components config (see ssh-key)